aws/resource_aws_wafv2_web_acl_logging_configuration.go

```diff
@@ -37,6 +38,78 @@ func resourceAwsWafv2WebACLLoggingConfiguration() *schema.Resource {
 				},
 				Description: "AWS Kinesis Firehose Delivery Stream ARNs",
 			},
+			"logging_filter": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"default_behavior": {
+							Type:         schema.TypeString,
+							Required:     true,
+							ValidateFunc: validation.StringInSlice([]string{"KEEP", "DROP"}, false),
```

Comment:
Nit: We can also use the constants here 👍 

```suggestion
							ValidateFunc: validation.StringInSlice(wafv2.FilterBehavior_Values(), false),
```